crates/pop-parachains/src/templates.rs

```diff
@@ -141,64 +155,75 @@ pub enum Error {
 #[cfg(test)]
 mod tests {
 	use super::*;
-	use std::str::FromStr;
+	use std::{collections::HashMap, str::FromStr};
+
+	fn templates_names() -> HashMap<String, Template> {
+		HashMap::from([
+			("standard".to_string(), Template::Standard),
+			("assets".to_string(), Template::Assets),
+			("contracts".to_string(), Template::Contracts),
+			("evm".to_string(), Template::EVM),
+			("cpt".to_string(), Template::ParityContracts),
+			("fpt".to_string(), Template::ParityFPT),
+		])
+	}
+	fn templates_urls() -> HashMap<String, &'static str> {
+		HashMap::from([
+			("standard".to_string(), "https://github.com/r0gue-io/base-parachain"),
```

Comment:
Feels like `base-parachain` should be renamed as well; Not related to the PR but should be addressed on that repo.

Reply:
For this suggestion is better you open an issue in the base-parachain repo.